core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -75,4 +75,16 @@ public String exportJson() {
         }
         return noticesByType;
     }
+
+    /**
+     * Adds all notices from another container.
+     * <p>
+     * This is useful for multithreaded validation: each thread has its own notice container which is merged
```

Comment:
I suggest adding a comment at the top of the class explaining the general lack of thread safety in this class, so later someone doesn't see the word "multithreaded" and start synchronizing everything.

Reply:
Done.